### PR TITLE
Added cap_add to the documentation for capabilities for easier search

### PR DESCRIPTION
**Pull request by RichardJActon**
##### SUMMARY

Added `cap_add` to the documentation for `capabilities` to make it easier to find the capabilities option if searching based on the docker name for this parameter.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docker container

### CODE REVIEW

File: plugins/modules/docker_container.py [hunk 1]
```diff
@@ -38,7 +38,7 @@
     type: int
   capabilities:
     description:
-      - List of capabilities to add to the container.
+      - List of capabilities to add to the container (equivalent to docker cap_add).
```
Comment (felixfontein) [inline plugins/modules/docker_container.py:41]: What exactly do you mean with `docker cap_add`? That's apparently not a CLI command (also the CLI option is called `--cap-add`).

Comment (RichardJActon) [reply plugins/modules/docker_container.py:41]: Ah yes looks like I'm conflating the docker-cli & docker-compose syntax. It's `cap_add` in compose for example as used here: https://hub.docker.com/r/pihole/pihole.

Would this be preferable: 

(equivalent to `docker --cap-add`, or `docker-compose` option `cap_add`).

Comment (felixfontein) [reply plugins/modules/docker_container.py:41]: I would mention `docker run --cap-add`, something like this:
```suggestion
      - List of capabilities to add to the container.
      - This is equivalent to C(docker run --cap-add).
```

Comment (RichardJActon) [reply plugins/modules/docker_container.py:41]: This is fine except that the original reason I discovered that this parameter was named differently from docker cli or compose was that when I searched the docs for the literal string `cap_add` whilst trying to translate a docker-compose file to an ansible playbook I didn't get a hit. I suspect others trying to switch to ansible might take the same path so it seems like a good idea to me to have the names of both the cli and compose equivalents in the docs. I'm new here so I don't know if the community has existing conventions around this and having both might get a bit bloated but I'd make it something that has the literal strings from both cli and compose e.g. 

`- This is equivalent to C(docker run --cap-add), or the C(docker-compose) option C(cap_add).`

Happy to go with your suggestion if you think this is still to vague and making it more specific about the compose equivalent would be to verbose.

Comment (felixfontein) [reply plugins/modules/docker_container.py:41]: How about: `- This is equivalent to C(docker run --cap-add), or the docker-compose option C(cap_add).`?

Comment (felixfontein) [reply plugins/modules/docker_container.py:41]: You still need to remove the part from this line :)
```suggestion
      - List of capabilities to add to the container.
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@RichardJActon thanks a lot for improving the docs!

**Pull request merged**